src/Middleware/Session/src/DistributedSession.cs

```diff
@@ -434,4 +437,13 @@ private static byte[] ReadBytes(Stream stream, int count)
         }
         return output;
     }
+
+    private sealed class DistributedSessionDebugView(DistributedSession session)
+    {
+        private readonly DistributedSession _session = session;
+
+        public bool IsAvailable => _session.IsAvailable;
+        public string Id => _session.Id;
+        public IEnumerable<string> Keys => _session.Keys.ToArray();
```

Comment:
I believe array has a different way of displaying. Something like `string[10]`. Change to `List<string>`? That provides `Count = 10`
```suggestion
        public IEnumerable<string> Keys => new List<string>(_session.Keys);
```